com.jianguoyun.Nutstore.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: com.jianguoyun.Nutstore
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+command: nutstore
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # So that we can sync files anywhere we want.
+  - --filesystem=home
+separate-locales: false
+
+modules:
+  # Dependency of webkitgtk.
+  - name: unifdef
+    buildsystem: simple
+    build-commands:
+      - make
+      - make prefix=/app install
+    sources:
+      - type: archive
+        url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+        sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+  - name: libsoup2
+    buildsystem: meson
+    config-opts:
+      - -Dtests=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz
+        sha256: e4b77c41cfc4c8c5a035fcdc320c7bc6cfb75ef7c5a034153df1413fa1d92f13
```

Comment:
libsoup2 should be from shared-modules https://github.com/flathub/shared-modules

Reply:
Done.